docs/tutorials/12_InformationFilterTutorial.py

```diff
@@ -0,0 +1,268 @@
+from stonesoup.types.detection import Detection
+from stonesoup.models.measurement.linear import LinearGaussian
+
+measurement_model = LinearGaussian(ndim_state = 4, # Number of state dimensions (position and velocity in 2D)
+                                  mapping=(0,2), # Mapping measurement vector index to state index
+                                  noise_covar=np.array([[5,0], # Covariance matrix for Gaussian PDF
+                                                      [0,5]])
+                                  )
+
+# taking measurements
+
+measurements = []
+
+for state in truth:
+    measurement = measurement_model.function(state,noise=True)
+    measurements.append(Detection(measurement,timestamp=state.timestamp,measurement_model = measurement_model))
+
+
+# %%
+# We plot the measurements using the Plotter class in Stone Soup. Again specifying the :math:`x`, :math:`y` position
+# indicies from the state vector.
+
+
+plotter.plot_measurements(measurements,[0,2])
+plotter.fig
+
+# %%
+# Running the Information Filter:
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# We must first import the :class:`~.InformationKalmanPredictor`, and :class:`~.InformationKalmanUpdater` from the
+# corresponding libraries.
+#
+# As before, the Predictor must be passed a Transition model, and the updater a Measurement model.
+# 
+# It is important to note that, unlike the Kalman Filter in Tutorial 1, the Information Filter requires
+# the prior estimate to be in the form of an :class:`~.InformationState` (not a :class:`~.GaussianState`).
+# 
+# The InformationState can be imported from stonesoup.types.state, and takes arguments: Information
+# state, Information Matrix and a timestamp.
+# 
+# The information matrix is defined as : :math:`{Y}_{k-1}` = :math:`[{P}_{k-1}]^{-1}`.
+# That is, the inverse of the covariance matrix.
+# The information state is defined as : :math:`{y}_{k-1}` = :math:`[{P}_{k-1}]^{-1}` :math:`{x}_{k-1}`.
+#
+# That is the matrix multiplication of the information matrix and the prior state in :math:`{x}_{k-1}`.
+# 
+# Using the same prior state as the original Kalman filtering example, we must firstly convert the
+# covariance to be the Information matrix, :math:`{Y}_{k-1}`, and calculate
+# the :class:`~.InformationState`, :math:`{y}_{k-1}`.
+#
+
+
+# predicting and updating
+from stonesoup.predictor.information import InformationKalmanPredictor
+from stonesoup.updater.information import InformationKalmanUpdater
+
+
+# Creating Information predictor and updater objects
+predictor = InformationKalmanPredictor(transition_model)
+updater = InformationKalmanUpdater(measurement_model)
+
+# %%
+# As before, we use the :class:`~.SingleHypothesis` class. The explicitly associates a single
+# predicted state to a single detection.
+
+from stonesoup.types.state import InformationState
+
+# Precision matrix - the inverse of the covariance matrix
+covar = np.diag([1.5, 0.5, 1.5, 0.5])
+precision_matrix = np.linalg.inv(covar) # information matrix
```

Comment:
They're actually the same thing. I'll add 'information matrix (precision matrix)' in place of the above comment to make it more coherent. Thank you.

Reply:
I'd prefer precision matrix to avoid confusion with the matrix forms of Shannon and Fisher information (both of which are often referred to as information matrices). Our great sentience (i.e. wikipedia) also reckons it's precision: https://en.wikipedia.org/wiki/Covariance_matrix#Inverse_of_the_covariance_matrix

Reply:
Thanks Jordi. That's great, I'll change it to 'precision matrix'. Can't dispute wikipedia!